doc/Exploratory-tests.md

```diff
@@ -0,0 +1,188 @@
+
+Test results:
+    The road is visible, always zoomed in to fill the entire screen. The point is following our position on the map.
+    Successful.
+
+### 3. test
+
+Test title: Log-out testing
+
+Test objective: Testing whether logging out works properly.
+
+Test steps:
+
+- After logging in with Google-account
+- Click on Home tab on the bottom of the screen
+- Click on Log out on the bottom left of the screen. This takes you back to the log-in page
+- Exit the application.
+- Start the application.
+
+Test results:
+
+- The user is logged-out after restarting the application. Successful
+
+<img src="assets/exploratory-tests/alreadyloggedin.png" alt="Already logged in.">
+
+<img src="assets/exploratory-tests/login.png" alt="Log-in screen.">
+
+### 4. test
+
+Test title: Starting app after logging in
+
+Test objective: Testing how the application reacts to starting after logging in.
+
+Test steps:
+
+- Log in with Google-account
+- Exit the application
+- Start the application
+
+Test result:
+
+- Anomaly detected. There is a brief moment when the log-in screen is displayed despite being logged-in. This is presumably caused due to slow cpu loading time. The button can be clicked, but does not crash the application.
+
+### 5. test
+
+Test title: Force-stopping the application
+
+Test objective: Testing how sessions react to force-stopping the application.
+
+Test steps:
+
+- Log in with Google-account
+- Start session
+- Exit application
+- Enter phone settings
+- Search for Jay
+- Click on Jay app (illustration how it looks like in emulator)
+- Click on Force stop
+- Start application
+- Click on Session on the bottom tab
+- Scroll down and click the last session
+
+<img src="assets/exploratory-tests/5appinfo.png" alt="Jay App info">
+
+<img src="assets/exploratory-tests/5forcestop.png" alt="Force stopping app">
+
+Test result:
+
+- Anomaly detected. Session keeps running after force-stopping the application, but the location is not refreshed.
+Proposition to improve: Store in a settings file whether a user is logged-in and display fragments accordingly.
+
+<img src="assets/exploratory-tests/sessionstillactive.png" alt="Session is still active">
+
+### 6. test
+
+Test title: Changing location to irreal distances from emulator settings
+
+Test objective: Testing whether the app will recognize if a user is moving unrealistically.
+
+Test steps:
+
+- Start application with emulator
+- Log in with Google-account
+- Start tracking service
+- Go to emulator settings
+
+<img src="assets/exploratory-tests/6emulatorsettings.png" alt="Emulator settings">
+
+- Set the location on the Single points tab (you can select a new location by searching for a place and clicking Set location on the bottom left of the screen)
+
+<img src="assets/exploratory-tests/singlepoint.png" alt="Single point option">
+
+- Switch back to the map display
+Test result:
```

Comment:
Illessz be néhány newline-t a Test result fölé és alá, máshol is jelen van ez a probléma.